### ISSUE TITLE
SearchableBy with nested relations

### ISSUE THREAD

**Issue by twdnhfr** [tailflow/laravel-orion#46]
Hey, 

is there a way to archive searching on related models (nested)? 

I have a User, Location and a Address Model and Iam trying to make the users name and the postalcode searchable:

```
protected function alwaysIncludes() : array
{
    return ['user', 'user.location', 'user.location.address'];
}

protected function searchableBy() : array
{
    return ['user.name', 'user.location.address.postalcode'];
}
```

But this will result in an Error:
Call to undefined method App\Models\User::postalcode()

whats the correct way to search on nested models?

**Comment by alexzarbn** [comment]
Hi @twdnhfr,

Currently it is only possible to use keyword-based search on relations one-level deep, e.g.:

```php
protected function searchableBy() : array
{
    return ['user.name'];
}
```

I will look into providing the support for nested relations in keyword-based search 👌

**Comment by alexzarbn** [comment]
The feature is now available as part of the [v2.0 release](https://github.com/tailflow/laravel-orion/releases/tag/2.0.0) 🎉
